### ISSUE TITLE
Fail to pip install pysdfgen in github action

### ISSUE THREAD

**Issue by HiroIshida** [iory/pySDFGen#14]
## bug description
I found that from yesterday, pysdfgen is not correctly installed in the github action. I found this error when installing package depending scikit-robot. This is bit problematic because all CI depends on scikit robot fails. To reproduce error, I created minimum repository with github action https://github.com/HiroIshida/reproduce_pip_error, and checked that pip install failed as expected:
```
Collecting pysdfgen
  Downloading pysdfgen-0.1.5.tar.gz (20 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line [34](https://github.com/HiroIshida/reproduce_pip_error/runs/7464800689?check_suite_focus=true#step:5:35), in <module>
        File "/tmp/pip-install-787f04wi/pysdfgen_93826457b7ca42c189b51b0f0eb869b1/setup.py", line 99, in <module>
          main()
        File "/tmp/pip-install-787f04wi/pysdfgen_93826457b7ca42c189b51b0f0eb869b1/setup.py", line 92, in main
          get_or_install('scikit-build')
        File "/tmp/pip-install-787f04wi/pysdfgen_93826457b7ca42c189b51b0f0eb869b1/setup.py", line 76, in get_or_install
          js_packages = json.loads(
        File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/json/__init__.py", line [35](https://github.com/HiroIshida/reproduce_pip_error/runs/7464800689?check_suite_focus=true#step:5:36)7, in loads
          return _default_decoder.decode(s)
        File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/json/decoder.py", line 3[40](https://github.com/HiroIshida/reproduce_pip_error/runs/7464800689?check_suite_focus=true#step:5:41), in decode
          raise JSONDecodeError("Extra data", s, end)
      json.decoder.JSONDecodeError: Extra data: line 3 column 1 (char 297)
      [end of output]
```
full log is available at https://github.com/HiroIshida/reproduce_pip_error/runs/7464800689?check_suite_focus=true

## versions 
For your information output of each package version is as follows: 
https://github.com/HiroIshida/reproduce_pip_error/blob/fb38b35aea0a73c144f7503ccd836f5ba8843c79/.github/workflows/test.yaml#L27

and the result is 
```
Name: setuptools
Version: 56.0.0
Summary: Easily download, build, install, upgrade, and uninstall Python packages
Home-page: https://github.com/pypa/setuptools
Author: Python Packaging Authority
Author-email: distutils-sig@python.org
License: UNKNOWN
Location: /opt/hostedtoolcache/Python/3.8.[13](https://github.com/HiroIshida/reproduce_pip_error/runs/7464800689?check_suite_focus=true#step:4:14)/x64/lib/python3.8/site-packages
Requires: 
Required-by: 
pip 22.1.2 from /opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/pip (python 3.8)
```
https://github.com/HiroIshida/reproduce_pip_error

**Comment by HiroIshida** [comment]
I think I can fix it. Please give me some time.

**Comment by HiroIshida** [comment]
The reason seems that the output of 
https://github.com/iory/pySDFGen/blob/1fea15dbcc82afe6f2fdaf91eebc39cf1bff3af8/setup.py#L78
starts to contain extra strings 
```
[notice] A new release of pip available: 22.1.2 -> 22.2
[notice] To update, run: pip install --upgrade pip
``` 
after recent update of pip, i guess. 

```
[{"name": "actionlib", "version": "1.13.2"}, {"name": "alabaster", "version": "0.7.12"}, {"name": "albumentations", "version": "1.2.0"}, {"name": "angles", "version": "1.9.13"}, {"name": "app-manager", "version": "1.3.0"}, {"name": "arhmm", "version": "0.0.1", "editable_project_location": "/home/h-ishida/python/arhmm"}, {"name": "asttokens", "version": "2.0.5"}, {"name": "autoflake", "version": "1.4"}, {"name": "Babel", "version": "2.10.3"}, {"name": "backcall", "version": "0.2.0"}, {"name": "black", "version": "22.3.0"}, {"name": "bondpy", "version": "1.8.6"}, {"name": "bunsetsu", "version": "0.0.1", "editable_project_location": "/home/h-ishida/python/bunsetsu"}, {"name": "cached-property", "version": "1.5.2"}, {"name": "camera-calibration", "version": "1.16.0"}, {"name": "camera-calibration-parsers", "version": "1.12.0"}, {"name": "catkin", "version": "0.8.10"}, {"name": "click", "version": "8.1.3"}, {"name": "community", "version": "1.0.0b1"}, {"name": "cv-bridge", "version": "1.16.0"}, {"name": "cvxopt", "version": "1.3.0"}, {"name": "dataclasses", "version": "0.6"}, {"name": "demon", "version": "2.0.6"}, {"name": "diagnostic-analysis", "version": "1.11.0"}, {"name": "diagnostic-common-diagnostics", "version": "1.11.0"}, {"name": "diagnostic-updater", "version": "1.11.0"}, {"name": "distro", "version": "1.7.0"}, {"name": "dynamic-reconfigure", "version": "1.7.2"}, {"name": "executing", "version": "0.8.3"}, {"name": "fingertip_pressure", "version": "1.9.0"}, {"name": "flake8", "version": "4.0.1"}, {"name": "gazebo_plugins", "version": "2.9.2"}, {"name": "gazebo_ros", "version": "2.9.2"}, {"name": "gencpp", "version": "0.6.5"}, {"name": "geneus", "version": "3.0.0"}, {"name": "genlisp", "version": "0.4.18"}, {"name": "genmsg", "version": "0.5.16"}, {"name": "gennodejs", "version": "2.0.2"}, {"name": "genpy", "version": "0.6.15"}, {"name": "gensim", "version": "4.2.0"}, {"name": "heck", "version": "0.0.1", "editable_project_location": "/home/h-ishida/misc/snippets/python/std_examples/cli"}, {"name": "image-geometry", "version": "1.16.0"}, {"name": "imageio", "version": "2.19.3"}, {"name": "imageio-ffmpeg", "version": "0.4.7"}, {"name": "imagesize", "version": "1.3.0"}, {"name": "importlib-metadata", "version": "4.11.4"}, {"name": "iniconfig", "version": "1.1.1"}, {"name": "interactive-markers", "version": "1.12.0"}, {"name": "ipython", "version": "8.4.0"}, {"name": "isort", "version": "5.10.1"}, {"name": "jedi", "version": "0.18.1"}, {"name": "joint-state-publisher", "version": "1.15.1"}, {"name": "joint-state-publisher-gui", "version": "1.15.1"}, {"name": "joint_trajectory_action_tools", "version": "0.0.12"}, {"name": "jsk_network_tools", "version": "2.2.11"}, {"name": "jsk_tools", "version": "2.2.11"}, {"name": "jsk_topic_tools", "version": "2.2.11"}, {"name": "karateclub", "version": "1.3.0"}, {"name": "laser_geometry", "version": "1.6.7"}, {"name": "matplotlib", "version": "3.3.4"}, {"name": "matplotlib-inline", "version": "0.1.3"}, {"name": "message-filters", "version": "1.15.14"}, {"name": "mohou", "version": "0.2.9", "editable_project_location": "/home/h-ishida/python/mohou"}, {"name": "mohou-contrib", "version": "0.0.11", "editable_project_location": "/home/h-ishida/python/mohou_contrib"}, {"name": "mohou-ros-utils", "version": "0.0.1", "editable_project_location": "/home/h-ishida/python/mohou_ros_utils"}, {"name": "moveit-core", "version": "1.1.9"}, {"name": "moviepy", "version": "1.0.3"}, {"name": "mypy", "version": "0.961"}, {"name": "mypy-extensions", "version": "0.4.3"}, {"name": "natsort", "version": "8.1.0"}, {"name": "numpy", "version": "1.23.0"}, {"name": "opencv-python", "version": "4.6.0.66"}, {"name": "opencv-python-headless", "version": "4.6.0.66"}, {"name": "ordered-set", "version": "4.1.0"}, {"name": "orjson", "version": "3.7.7"}, {"name": "packaging", "version": "21.3"}, {"name": "parso", "version": "0.8.3"}, {"name": "pathspec", "version": "0.9.0"}, {"name": "pickleshare", "version": "0.7.5"}, {"name": "pip", "version": "22.1.2"}, {"name": "platformdirs", "version": "2.5.2"}, {"name": "pr2_camera_synchronizer", "version": "1.6.32"}, {"name": "pr2_computer_monitor", "version": "1.6.32"}, {"name": "pr2_controller_manager", "version": "1.8.20"}, {"name": "pr2_counterbalance_check", "version": "1.0.15"}, {"name": "pr2_power_board", "version": "1.1.10"}, {"name": "pr2-tuck-arms-action", "version": "0.0.12"}, {"name": "proglog", "version": "0.1.10"}, {"name": "prompt-toolkit", "version": "3.0.29"}, {"name": "pudb", "version": "2022.1.1"}, {"name": "pure-eval", "version": "0.2.2"}, {"name": "py", "version": "1.11.0"}, {"name": "pybullet", "version": "3.2.5"}, {"name": "pycodestyle", "version": "2.8.0"}, {"name": "pycollada", "version": "0.7.2"}, {"name": "pydantic", "version": "1.9.1"}, {"name": "pyflakes", "version": "2.4.0"}, {"name": "pyglet", "version": "1.5.26"}, {"name": "Pygments", "version": "2.12.0"}, {"name": "PyGSP", "version": "0.5.1"}, {"name": "pyinstrument", "version": "4.2.0"}, {"name": "pyparsing", "version": "3.0.9"}, {"name": "pytest", "version": "7.1.2"}, {"name": "python-fcl", "version": "0.6.1"}, {"name": "python-Levenshtein", "version": "0.12.2"}, {"name": "python-louvain", "version": "0.16"}, {"name": "python-qt-binding", "version": "0.4.4"}, {"name": "qt-dotgraph", "version": "0.4.2"}, {"name": "qt-gui", "version": "0.4.2"}, {"name": "qt-gui-cpp", "version": "0.4.2"}, {"name": "qt-gui-py-common", "version": "0.4.2"}, {"name": "quadprog", "version": "0.1.11"}, {"name": "qudida", "version": "0.0.4"}, {"name": "resource_retriever", "version": "1.12.7"}, {"name": "rosbag", "version": "1.15.14"}, {"name": "rosboost-cfg", "version": "1.15.8"}, {"name": "rosclean", "version": "1.15.8"}, {"name": "roscreate", "version": "1.15.8"}, {"name": "rosgraph", "version": "1.15.14"}, {"name": "roslaunch", "version": "1.15.14"}, {"name": "roslib", "version": "1.15.8"}, {"name": "roslint", "version": "0.12.0"}, {"name": "roslz4", "version": "1.15.14"}, {"name": "rosmake", "version": "1.15.8"}, {"name": "rosmaster", "version": "1.15.14"}, {"name": "rosmsg", "version": "1.15.14"}, {"name": "rosnode", "version": "1.15.14"}, {"name": "rosparam", "version": "1.15.14"}, {"name": "rospy", "version": "1.15.14"}, {"name": "rosservice", "version": "1.15.14"}, {"name": "rostest", "version": "1.15.14"}, {"name": "rostopic", "version": "1.15.14"}, {"name": "rosunit", "version": "1.15.8"}, {"name": "roswtf", "version": "1.15.14"}, {"name": "rqt_action", "version": "0.4.9"}, {"name": "rqt_bag", "version": "0.5.1"}, {"name": "rqt_bag_plugins", "version": "0.5.1"}, {"name": "rqt_console", "version": "0.4.11"}, {"name": "rqt_dep", "version": "0.4.12"}, {"name": "rqt_graph", "version": "0.4.14"}, {"name": "rqt_gui", "version": "0.5.3"}, {"name": "rqt_gui_py", "version": "0.5.3"}, {"name": "rqt_image_view", "version": "0.4.16"}, {"name": "rqt_launch", "version": "0.4.9"}, {"name": "rqt_logger_level", "version": "0.4.11"}, {"name": "rqt-moveit", "version": "0.5.10"}, {"name": "rqt_msg", "version": "0.4.10"}, {"name": "rqt_nav_view", "version": "0.5.7"}, {"name": "rqt_plot", "version": "0.4.13"}, {"name": "rqt_pose_view", "version": "0.5.11"}, {"name": "rqt_publisher", "version": "0.4.10"}, {"name": "rqt_py_common", "version": "0.5.3"}, {"name": "rqt_py_console", "version": "0.4.10"}, {"name": "rqt-reconfigure", "version": "0.5.5"}, {"name": "rqt-robot-dashboard", "version": "0.5.8"}, {"name": "rqt-robot-monitor", "version": "0.5.14"}, {"name": "rqt_robot_steering", "version": "0.5.12"}, {"name": "rqt_runtime_monitor", "version": "0.5.9"}, {"name": "rqt-rviz", "version": "0.7.0"}, {"name": "rqt_service_caller", "version": "0.4.10"}, {"name": "rqt_shell", "version": "0.4.11"}, {"name": "rqt_srv", "version": "0.4.9"}, {"name": "rqt_tf_tree", "version": "0.6.3"}, {"name": "rqt_top", "version": "0.4.10"}, {"name": "rqt_topic", "version": "0.4.13"}, {"name": "rqt_web", "version": "0.4.10"}, {"name": "rviz", "version": "1.14.14"}, {"name": "scikit-build", "version": "0.15.0"}, {"name": "scikit-robot", "version": "0.0.16", "editable_project_location": "/home/h-ishida/python/scikit-robot"}, {"name": "sensor-msgs", "version": "1.13.1"}, {"name": "setuptools", "version": "56.0.0"}, {"name": "sklearn", "version": "0.0"}, {"name": "smach", "version": "2.5.0"}, {"name": "smach-ros", "version": "2.5.0"}, {"name": "smart-open", "version": "6.0.0"}, {"name": "smclib", "version": "1.8.6"}, {"name": "sound-play", "version": "0.3.13"}, {"name": "Sphinx", "version": "5.0.2"}, {"name": "sphinx-rtd-theme", "version": "1.0.0"}, {"name": "sphinxcontrib-applehelp", "version": "1.0.2"}, {"name": "sphinxcontrib-devhelp", "version": "1.0.2"}, {"name": "sphinxcontrib-htmlhelp", "version": "2.0.0"}, {"name": "sphinxcontrib-jsmath", "version": "1.0.1"}, {"name": "sphinxcontrib-qthelp", "version": "1.0.3"}, {"name": "sphinxcontrib-serializinghtml", "version": "1.1.5"}, {"name": "srdfdom", "version": "0.6.3"}, {"name": "stack-data", "version": "0.3.0"}, {"name": "tf", "version": "1.13.2"}, {"name": "tf-conversions", "version": "1.13.2"}, {"name": "tf2-geometry-msgs", "version": "0.7.5"}, {"name": "tf2-kdl", "version": "0.7.5"}, {"name": "tf2-py", "version": "0.7.5"}, {"name": "tf2-ros", "version": "0.7.5"}, {"name": "tinyfk", "version": "0.4.8"}, {"name": "tomli", "version": "2.0.1"}, {"name": "topic-tools", "version": "1.15.14"}, {"name": "torch", "version": "1.11.0"}, {"name": "torchvision", "version": "0.12.0"}, {"name": "traitlets", "version": "5.3.0"}, {"name": "trimesh", "version": "3.12.6"}, {"name": "tscom", "version": "0.0.1", "editable_project_location": "/home/h-ishida/python/tscom"}, {"name": "tunable-filter", "version": "0.0.3", "editable_project_location": "/home/h-ishida/python/tunable_image_filter"}, {"name": "types-PyYAML", "version": "6.0.8"}, {"name": "typing_extensions", "version": "4.2.0"}, {"name": "urdfdom-py", "version": "0.4.6"}, {"name": "urwid", "version": "2.1.2"}, {"name": "urwid-readline", "version": "0.13"}, {"name": "wheel", "version": "0.37.1"}, {"name": "xacro", "version": "1.14.13"}]

[notice] A new release of pip available: 22.1.2 -> 22.2
[notice] To update, run: pip install --upgrade pip
```
